src/main/java/kr/co/onehunnit/onhunnit/service/NotificationService.java

```diff
@@ -0,0 +1,87 @@
+package kr.co.onehunnit.onhunnit.service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import io.swagger.v3.oas.annotations.tags.Tag;
+import kr.co.onehunnit.onhunnit.config.exception.ApiException;
+import kr.co.onehunnit.onhunnit.config.exception.ErrorCode;
+import kr.co.onehunnit.onhunnit.domain.account.Account;
+import kr.co.onehunnit.onhunnit.domain.notification.Notification;
+import kr.co.onehunnit.onhunnit.dto.notification.NotificationRequestDto;
+import kr.co.onehunnit.onhunnit.dto.notification.NotificationResponseDto;
+import kr.co.onehunnit.onhunnit.repository.NotificationRepository;
+import lombok.RequiredArgsConstructor;
+import reactor.core.publisher.Mono;
+
+@RequiredArgsConstructor
+@Service
+public class NotificationService {
+
+	private final WebClient.Builder webClientBuilder;
+	private final AccountService accountService;
+	private final NotificationRepository notificationRepository;
+
+	private static final String EXPO_BACKEND_URI = "https://exp.host/--/api/v2/push/send";
+
+	public String pushNotification(String accessToken, NotificationRequestDto notificationRequestDto) {
+		Account account = accountService.getAccountByToken(accessToken);
+		WebClient webClient = webClientBuilder.build();
+
+		Mono<String> response = webClient.post()
+			.uri(EXPO_BACKEND_URI)
+			.bodyValue(notificationRequestDto)
+			.retrieve()
+			.onStatus(status -> status.isError(), clientResponse -> {
+				return Mono.error(new RuntimeException("푸시 알림 전송에 실패하였습니다."));
+			})
+			.bodyToMono(String.class);
+
+		Notification notification = Notification.builder()
+			.title(notificationRequestDto.getTitle())
+			.body(notificationRequestDto.getBody())
+			.account(account)
+			.build();
+
+		notificationRepository.save(notification);
+		return response.block();
```

Comment:
이런 느낌으로 알림 보낸 거 성공하고 저장하는게 좋을 것 같네요.
이 코드가 맞는지는 잘 모르겠지만요

```suggestion
		return response.doOnSuccess(res -> notificationRepository.save(notification));
```

Reply:
수정완료했습니다~